artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireConnection.java

```diff
@@ -1338,7 +1339,11 @@ public Response processRemoveSubscription(RemoveSubscriptionInfo subInfo) throws
 
       @Override
       public Response processRollbackTransaction(TransactionInfo info) throws Exception {
-         Transaction tx = lookupTX(info.getTransactionId(), null, true);
+         Transaction tx = lookupTX(info.getTransactionId(), null);
+
+         if (tx == null) {
+            throw new IllegalStateException("Transaction not started, " + info.getTransactionId());
```

Comment:
I would use a logger-id from the MessageBundle.

Reply:
there is currently no use of the bundle in any of the exceptions, that can be another PR if really necessary